src/api/PostList/PostList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CommentList from '../CommentList/CommentList';
+import User from '../User/User';
+
+const PostList = ({ posts }) => (
+  posts.map(post => (
+    <>
+      <div className="post-title">
+        {post.title}
+      </div>
+      <div>
+        {post.body}
+      </div>
+
+      <div className="user">
+        <User {...post.user} key={post.id} />
+      </div>
+
+      <div>
+        <CommentList comments={post.comments} />
+      </div>
+    </>
+  ))
```

Comment:
Again, same thing here.
Need parent tag and each post should be rendered in own `Post` component.
`Post` should include `User` and `CommentList` components.